__tests__/app.test.js

```diff
@@ -40,13 +40,52 @@ describe("GET /api/topics", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id", () => {
+  test("Status: 200", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+  test("Status: 200, returns an object with corresponding keys", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toEqual(
+          expect.objectContaining({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+          })
```

Comment:
This test should be more specific, are you expecting any or specific values? How can you be sure this is the right article?